From 23c56523148780c204709b934801afa612a75763 Mon Sep 17 00:00:00 2001
From: Paul Harrison <pharrison@mythtv.org>
Date: Tue, 25 Mar 2014 22:41:57 +0000
Subject: [PATCH 1/3] mythutil: send a MUSIC_ALBUMART_CHANGED message in
 ExtractImage

This changes the message on successfully extracting an image from a tracks tag
and saving it to the cache to MUSIC_ALBUMART_CHANGED. Sending a
MUSIC_METADATA_CHANGED message can result in an infinite loop under some
circumstances.
---
 mythtv/programs/mythutil/musicmetautils.cpp |    5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/mythtv/programs/mythutil/musicmetautils.cpp b/mythtv/programs/mythutil/musicmetautils.cpp
index f2d9976..942c64c 100644
--- a/mythtv/programs/mythutil/musicmetautils.cpp
+++ b/mythtv/programs/mythutil/musicmetautils.cpp
@@ -168,9 +168,8 @@ static int ExtractImage(const MythUtilCommandLineParser &cmdline)
 
     delete tagger;
 
-    // tell any clients that the metadata for this track has changed
-    // TODO check we need this
-    gCoreContext->SendMessage(QString("MUSIC_METADATA_CHANGED %1").arg(songID));
+    // tell any clients that the albumart for this track has changed
+    gCoreContext->SendMessage(QString("MUSIC_ALBUMART_CHANGED %1 %2").arg(songID).arg(type));
 
     return GENERIC_EXIT_OK;
 }
-- 
1.7.10.2

